.github/workflows/iroha2-dev-pr.yml

```diff
@@ -3,11 +3,6 @@ name: I2::Dev::Tests
 on:
   pull_request:
     branches: [iroha2-dev]
-    paths:
-      - '**.rs'
-      - '**.json'
-      - '**.toml'
-      - '.github/workflows/**.yml'
```

Comment:
this means that workflow is only run when some of these files are modified? You think we don't need this?

Reply:
I'm not sure, I wouldn't change it in this PR